<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>CHAPTER 2</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="customstyles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">M. Lott</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Academic &amp; Professional Development
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="./academic_progress.html">Academic Progress</a>
    </li>
    <li>
      <a href="./professional_development.html">Professional Development</a>
    </li>
  </ul>
</li>
<li>
  <a href="./outreach.html">Outreach &amp; Service</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Dissertation Research
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="./intro.html">Introduction</a>
    </li>
    <li>
      <a href="./chapter_1.html">Chapter 1</a>
    </li>
    <li>
      <a href="./chapter_2.html">Chapter 2</a>
    </li>
    <li>
      <a href="./chapter_3.html">Chapter 3</a>
    </li>
  </ul>
</li>
<li>
  <a href="./additional_research.html">Additional Research</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://meganejlott.github.io/research_updates/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">CHAPTER 2</h1>

</div>


<div id="genomic-investigation-into-the-community-level-interactions-between-bacteria-and-algae-in-the-indian-river-lagoon-and-st.lucie-estuaries" class="section level2">
<h2>Genomic Investigation into the Community-Level Interactions between Bacteria and Algae in the Indian River Lagoon and St. Lucie Estuaries</h2>
<p>Megan Lott<sup>1</sup>, Michael Wetz<sup>2</sup>, Erin K. Lipp<sup>1</sup></p>
<p><sup>1</sup>Department of Environmental Health Sciences, University of Georgia; <sup>2</sup>Department of Life Sciences, Texas A&amp;M University at Corpus Christi</p>
<p>Target Journal: Frontiers in Microbiology (Impact Factor 4.26)</p>
</div>
<div id="motivation" class="section level1">
<h1>MOTIVATION</h1>
<p>Interactions between bacteria and algae have ecological implications for the productivity and stability of aquatic environments. Through their co-evolution, bacteria and phytoplankton have formed complex relationships, ranging from mutualism to competition, to optimize their acquisition of macro- and micro- nutrients. Research has demonstrated that bacteria play an important role in the biological control of algal blooms, and that their associations with phytoplankton are predominately species-specific. Despite decades of research exploring these associations, there is limited understanding of the underlying mechanisms and functional implications of bacterial-algal interactions in the natural environment.</p>
<p>Until recently, our understanding of bacterial-algal interactions has relied on traditional culture-based and microscopic methods. Laboratory observations have relied on unialgal or unibacterial cultures to tease apart associations that, in the natural environment, may be complex and influenced by multiple biotic and abiotic factors. Field studies have examined natural assemblages of bacterial and algal communities, but previous studies have been constrained to the resolution and scale of traditional or nonmolecular techniques. Coupled with bioinformatics and predictive modeling, scientists are utilizing emerging -omics technologies to revisit basic questions on bacterial-algal interactions. Next generation sequencing (NGS) methods can be utilized as a novel tool to examine community-level associations between bacteria and algae with higher resolution than traditional culture-based methods.</p>
<p>Investigating bacterial-algal interactions improves our understanding of the role bacteria play in shaping the ecology of phytoplankton. By identifying common partners in community assemblages in natural waters, we may be better positioned to ask questions about the functional relationships between these microorganisms.</p>
</div>
<div id="context" class="section level1">
<h1>CONTEXT</h1>
<p>Stretching across Florida’s east coast, the Indian River Lagoon (IRL) is an estuary of national significance and is protected by the USA National Estuaries Program. In recent years, the IRL has suffered from dense and damaging algal blooms, including those caused by the novel brown tide algae Aureoumbra lagunensis and by the toxic blue-green algae Microcystis aeruginosa.</p>
</div>
<div id="research-objectives" class="section level1">
<h1>RESEARCH OBJECTIVES</h1>
<p><strong>1. Utilize molecular methods to identify the correlation between bacterial and phytoplankton abundance in natural waters.</strong></p>
<p>We aim to resolve the relationships between bacterial and phytoplankton abundance by examining the correlations between traditional measures of abundance and biomass (microscopic counts, chlorophyll a and phaeopigments) and novel measures of abundance (total 16S and 18S abundance).</p>
<p><strong>2. Characterize the composition of planktonic and phytoplankton-associated bacterial communities in the IRL and SLE.</strong></p>
<p>We aim to identify symbiotic partners of bacteria and microalgae by examining species-level changes in the 16S and 18S microbial communities in the natural environment. We expect the particle-attached bacterial community to be specialized and significantly different than that of the free-living bacteria(Jonathan J. Cole 1982; Main et al. 2015)</p>
</div>
<div id="methods" class="section level1">
<h1>METHODS</h1>
<p><strong>Sampling Strategy:</strong> Water samples were collected in triplicate from three locations in both the IRL and the SLE. Samples were collected weekly between June 2019 and July 2019 (n = 144).</p>
<p>In the Northern Indian River Lagoon (IRL), samples were collected from three locations: Scottsmoor Landing (IRL 1), Titusville Pier (IRL 2), and the Beacon 42 Boat Ramp (IRL 3). Samples were collected on the following dates: 6/10, 6/17, 6/24, 7/1, 7/8, 7/15, 7/22, 7/29. The IRL is not heavily influenced by tidal mixing.</p>
<p>In the St. Lucie Estuary (SLE), samples were collected from three locations: Snug Harbor Yacht Club (SLE 1), Stuart Boardwalk (SLE 2), and Leighton Park (SLE 3). Samples were collected on the following dates: 6/5, 6/12, 6/19, 6/26, 7/3, 7/17, 7/24, 7/31. The SLE is tidally influenced. Samples were collected during the outgoing tide.</p>
<div class="figure">
<img src="chapter1/study_sites.png" alt="Figure 1. Sample Locations in the IRL and SLE" />
<p class="caption">Figure 1. Sample Locations in the IRL and SLE</p>
</div>
<p><strong>Sample Collection:</strong> Water samples were collected in triplicate and pre-filtered through 200 µm mesh to remove large debris and detritus. Samples were stored in 1 L sterile polypropylene bottles and stored on ice, in the dark.</p>
<p><strong>Chlorophyll a and Phaeopigments:</strong> Bulk water samples (25 mL) were filtered through 25 mm GF/F filters at low pressure (≤ 5 in Hg). Filters were stored in 5 mL polyethylene tubes at -20°C and store until analysis. Filters will be analyzed by EPA Method 445.0 using acetone extraction and fluorimeter.</p>
<p><strong>Quantification of Algal Abundance by Microscopy:</strong> Bulk water samples (5 mL) were fixed with 200 µL of acid Lugols in glass vials and stored at room temperature in the dark. Abundance of microalgae was quantified microscopically using a hemocytometer.</p>
<p><strong>Quantification of Algal and Bacterial Abundance by Flow Cytometry:</strong> Bulk water samples (5 mL) were fixed with glutaraldehyde at 1% and store at -80°C until analysis by flow cytometry.</p>
<p><strong>Collection of Sample DNA:</strong> Water samples were size fractionated in series onto polycarbonate filters (2. 0 µm) and Sterivex filters (0.22 µm). Filters were stored at -20°C until extraction.</p>
<p><strong>DNA Extraction:</strong> Sample DNA was extracted from stored polycarbonate filters (2.0 µm) using the Qiagen PowerSoil Pro Kit. DNA will be extracted from stored Sterivex filters (0.22 µm) using phenol:chloroform:isoamyl extractions.</p>
<p><strong>Next Generation Sequencing Methods:</strong> Bacterial 16S rRNA will be amplified using universal 16S primers 515F/806R(Caporaso et al. 2011) targeting the V4V5 hypervariable region. 18S rRNA will be amplified using the universal 18S primers EukA/329R(Gobler et al. 2013). Genomic libraries will be prepared with custom Illumina adapters with barcodes. The genomic libraries will be sequenced by the Georgia Genomics and Bioinformatics Core (GGBC), using Illumina MiSeq PE300. Sequence data will be analyzed in R using the DADA2 pipeline.</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
